outputs.tf

```diff
@@ -13,5 +13,5 @@ output "elasticache_replication_group" {
 
 output "elasticache_route53_record" {
```

Comment:
I think it's great to return the full record, but for convenience of the module user, we typically would add an entry for the actual fqdn. I don't think we should stop doing that so that people don't have to check documentation or output (which you only see post-run) for what the actual value they need is called.

Also think we should add the elasticache endpoint. You could coalesce the two so that you get the CNAME when `create_route_53_cname_record` is used and otherwise get the official endpoint. A single output that did this would be lovely IMO.

Reply:
:+1: done did it